lib/matplotlib/cbook/deprecation.py

```diff
@@ -221,9 +217,11 @@ def wrapper(*args, **kwargs):
 
         old_doc = textwrap.dedent(old_doc or '').strip('\n')
         message = message.strip()
-        new_doc = (('\n.. deprecated:: %(since)s'
-                    '\n    %(message)s\n\n' %
-                    {'since': since, 'message': message}) + old_doc)
+        new_doc = ('.. deprecated:: {}\n'
```

Comment:
I'd rather keep named placeholders, i.e. `{since}`.

When do we require Python 3.6 so that we can use f-strings? 😏 So long, I'd take the `.format(since=since, ...)` workaround.

Reply:
done